src/dash-table/components/Filter/Column.tsx

```diff
@@ -27,16 +23,6 @@ export default class ColumnFilter extends PureComponent<IColumnFilterProps, ICol
         };
     }
 
-    componentWillReceiveProps(nextProps: IColumnFilterProps) {
-        const { value: nextValue } = nextProps;
-
-        if (this.state.value !== nextValue) {
-            this.setState({
-                value: nextValue
-            });
-        }
-    }
-
```

Comment:
This was dead code -- dating back to before the IsolatedInput was added - nothing depends on the state here, removing.